packages/dx-react-grid-bootstrap4/src/templates/table-summary-item.test.jsx

```diff
@@ -27,10 +34,28 @@ describe('TableSummaryItem', () => {
 
   it('should pass rest props to the root element', () => {
     const tree = shallow((
-      <TableSummaryItem className="custom-class" />
+      <TableSummaryItem
+        {...defaultProps}
+        className="custom-class"
```

Comment:
I think, in this test we should check both classes: `dx-g-bs4-table-summary-item` and `custom-class`. Also, we can rename it to `should pass the className prop to the root element`.